orchestra/management/commands/migrate_certifications.py

```diff
@@ -0,0 +1,26 @@
+from django.core.management.base import BaseCommand
+from orchestra.workflow.certifications import migrate_certifications
+
+
+class Command(BaseCommand):
+    help = 'Migrates specified certifications across workflows.'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'workflow_src',
+            help='Slug for the source workflow from which to migrate '
+                 'certifications.')
+        parser.add_argument(
+            'workflow_dst',
+            help='Slug for the destination workflow to which certifications '
+                 'will be migrated.')
+        parser.add_argument(
+            '--certifications', nargs='+',
```

Comment:
it's usually nice to add a `metavar` (see https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_argument) to make the help string look nicer.